src/containers/find-offer/offer-card-square/OfferCardSquare.styles.ts

```diff
@@ -31,21 +57,32 @@ export const styles = {
     fontWeight: 600,
     color: 'primary.700'
   },
+  devider: {
+    width: '100%'
```

Comment:
you use width 100% a lot, similar to do alignItems stretch on parent

Reply:
> you use width 100% a lot, better to do alignItems stretch on parent

they were unnecessary, removed them 👍